.github/workflows/build.yml

```diff
@@ -12,89 +12,57 @@ jobs:
     container: gitlab-registry.cern.ch/acts/machines/ubuntu1910
     steps:
     - uses: actions/checkout@v1
-
     - name: Configure
       run: >
-        mkdir build && cd build
+        mkdir build
+        && cd build
         && cmake ..
         -DCMAKE_BUILD_TYPE=Release
-        -DCMAKE_CXX_STANDARD=17
```

Comment:
Do you specify the standard version explicitly somewhere else?

Reply:
It is already specified per target

Reply:
Btw, I still need to get the CI build running. The build configuration might still change.